apps/web/pages/auth/login.tsx

```diff
@@ -94,17 +103,22 @@ export default function Login({
     </Button>
   );
 
+  const communityLogin = queryParams.sso !== undefined && queryParams.sig !== undefined;
```

Comment:
We need to be able to know if there is an attempt to login to community to change a bit the sign in process below.